CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ### Version 5.1.0-alpha9
 
 - Add ARM64 support for windows [#2137](https://github.com/react-native-community/react-native-video/pull/2137)
+- Fix deprecated API bug for windows [#2119](https://github.com/react-native-video/react-native-video/pull/2119)
+- Added `rate` property and autolinking support for windows [#2206](https://github.com/react-native-video/react-native-video/pull/2206)
```

Comment:
should this be on a new version?

Reply:
`5.1.0-alpha8` is the latest version, `5.1.0-alpha9` should be the new version (which so far seems pretty windows focused).